pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -12,6 +12,7 @@ import (
 
 var (
 	sources = []string{Nvd, RedHat, Debian, DebianOVAL, Alpine, Amazon, OracleOVAL, SuseCVRF, Photon,
+		//sources = []string{Nvd, RedHat, Debian, DebianOVAL, Ubuntu, Alpine, Amazon, OracleOVAL, SuseCVRF, Photon,
```

Comment:
@knqyf263 why is `Ubuntu` missing from this list? You can see this test currently fails if it's not included https://github.com/aquasecurity/trivy-db/pull/48/files#diff-bc5f6ed1d12bdab4d1aeca18ca3003d3R112-R143

Reply:
I added it: https://github.com/aquasecurity/trivy-db/pull/48/commits/3d3689f41fc146e64a894fe9cdd472f6d46ae747